public_html/style.css

```diff
@@ -494,55 +575,54 @@ img {
 
 .cbailylaw__testimonial-grids {
     display: grid;
-    grid-template-columns: auto auto;
+    grid-template-columns: 50% 50%;
     gap: 50px;
 }
 
-.cbailylaw__testimonial-grid-full p,
 .cbailylaw__testimonial-grid p {
     font: 300 20px/34px var(--font-body);
     color: var(--color-black);
-    min-height: 170px;
     letter-spacing: 0.8px;
-    /* text-transform: capitalize; */
 }
 
 .cbailylaw__testimonial-grid.attribution {
```

Comment:
I don't think we're using this any more (I added this last week and it didn't work).